packages/app/features/Send/InformationInput/components/TransactionFee/TokenFeeDropDown.tsx

```diff
@@ -0,0 +1,77 @@
+import type { FC } from 'react';
+import { Image, StyleSheet } from 'react-native';
+import type { Token } from '@walless/core';
+import { Hoverable, modalActions, Text, View } from '@walless/gui';
+import type { TokenDocument } from '@walless/store';
+
+interface Props {
+	tokens: TokenDocument[];
+	onSelect: (token: Token) => void;
+	selectedToken?: Token;
+}
+
+const TokenFeeDropDown: FC<Props> = ({ tokens, onSelect, selectedToken }) => {
+	const handleSelectToken = (token: TokenDocument) => {
+		onSelect(token);
+		modalActions.destroy('NetworkFee');
+	};
+
+	const selectedStyle = {
+		backgroundColor: '#56667466',
+	};
+
+	return (
+		<View style={styles.dropdown}>
+			{tokens.map((token, idx) => {
+				const name = token.metadata?.symbol ?? 'Unknown';
+
+				return (
+					<Hoverable
+						style={[
+							styles.tokenOption,
+							token === selectedToken && selectedStyle,
+						]}
+						key={idx}
+						onPress={() => handleSelectToken(token)}
+					>
+						<Image
+							source={{
+								uri:
+									token.metadata?.imageUri ??
+									'/img/send-token/unknown-token.jpeg',
```

Comment:
carefully consider the value if you want to use nullish coalescing

Reply:
i handle this case for some tokens which are mint unofficially so that we cannot fetch their metadata on solana

Reply:
I mean make sure `imageUri` value is not set to empty string